lib/srv/regular/sshserver.go

```diff
@@ -1048,7 +1048,9 @@ func (s *Server) HandleRequest(ctx context.Context, r *ssh.Request) {
 	case teleport.KeepAliveReqType:
 		s.handleKeepAlive(r)
 	case teleport.RecordingProxyReqType:
-		s.handleRecordingProxy(r)
+		s.handleRecordingProxy(ctx, r)
```

Comment:
Can we also leave a `DELETE IN` comment for `handleRecordingProxy`?

If so, we should replace the last usage of `isRecordingProxy` in `proxy.PortForwardToNode`, so that v11 tsh will work with v12 server.